sae_lens/config.py

```diff
@@ -29,6 +29,11 @@
 
 HfDataset = DatasetDict | Dataset | IterableDatasetDict | IterableDataset
 
+ActivationNormalizationStrategy = Literal[
+    "none", "expected_average_only_in", "constant_norm_rescale", "layer_norm"
+]
```

Comment:
there were a few comments where the layer norm option was not noted. Though that centralising the definition would help